packages/proto/lib/index.js

```diff
@@ -1,2 +1,2 @@
-module.exports.coresdk = require('./coresdk').coresdk;
 module.exports = require('./temporal');
+module.exports.coresdk = require('./coresdk').coresdk;
```

Comment:
We should just unify the 2 generated files into 1.
The original reason that I split them was because the bundle side exploded when I included `temporal`.
This is no longer a restriction.

Reply:
Combined in https://github.com/temporalio/sdk-typescript/commit/cf12c4c1b7d2bde493c4ef7ae315b4f2dace2c52